_config/graphql.yml

```diff
@@ -0,0 +1,15 @@
+---
+Name: cmsgraphql
+---
+SilverStripe\GraphQL\Controller:
+  schema:
+    scaffolding:
+      types:
+        SilverStripe\CMS\Model\SiteTree:
+          fields: '*'
```

Comment:
Curious why this is wildcarded. Is it to support the "compare" view, where you'll show all the fields?

Any queries that depend on an all-inclusive field list will need to be generated dynamically, which is something we're still working on. Is that the idea?

Reply:
We only really need the versioning information at the moment, so wildcarding the fields is probably not required. I'll adjust.